app/src/main/java/com/example/madcamp_week1/PhotoAdapter.kt

```diff
@@ -29,7 +33,15 @@ class PhotoAdapter(val context:Context, val itemList: ArrayList<Photos>) :
         val photoText = itemView?.findViewById<LinearLayout>(R.id.choice_text)
 
         fun bind (photo: Photos, context: Context) {
-            photoImage?.setImageResource(photo.resId)
+            if (ContextCompat.checkSelfPermission(context, Manifest.permission.READ_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED)
+            {
+                photoImage?.setImageURI(photo.uri)
+            }
+            else
+            {
+                Toast.makeText(context,"갤러리 접근 권한을 허용해주세요.",Toast.LENGTH_LONG).show()
+            }
```

Comment:
Inconsistent brace wrapping styles.

```suggestion
            if (ContextCompat.checkSelfPermission(context, Manifest.permission.READ_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED)  {
                photoImage?.setImageURI(photo.uri)
            }
            else {
                Toast.makeText(context,"갤러리 접근 권한을 허용해주세요.",Toast.LENGTH_LONG).show()
            }
```